go/libraries/doltcore/sqle/rows.go

```diff
@@ -30,19 +31,19 @@ type doltTableRowIter struct {
 	sql.RowIter
 	table    *DoltTable
 	rowData  types.Map
-	ctx      *sql.Context
+	ctx      context.Context
```

Comment:
Everything in this package should use *sql.Context

Reply:
fixed